web/src/scripts/webflow/helpers.mjs

```diff
@@ -36,19 +38,19 @@ export const resolveApiPromises = async (promiseArray) => {
 };
 
 export const catchRateLimitErrorAndRetry = async (error, retryFunc, ...params) => {
-  console.debug("in Catch Rate and retry");
+  isNotTestEnv ?? console.debug("in Catch Rate and retry");
```

Comment:
curios, what value does adding this variable add? Do devs run code in a test environment? It will render in dev.